contracts/src/TeleporterTokenSource.sol

```diff
@@ -29,7 +28,7 @@ abstract contract TeleporterTokenSource is ITeleporterTokenBridge, TeleporterOwn
     bytes32 public immutable blockchainID;
 
     /// @notice The ERC20 token this contract uses to pay for Teleporter fees.
-    IERC20 public immutable feeToken;
+    address public immutable feeTokenAddress;
```

Comment:
Was this change necessary? All fee tokens should still implement the `IERC20` interface right?

Reply:
no this was not necessary, and yes it still implements `IERC20`. The change was made because this contract only interacts with the `address` type, and doesn't use `IERC20` functionality, so no need for extra conversion.